docs/get-started/introduction.md

```diff
@@ -38,7 +39,16 @@ NFD-Worker is a daemon responsible for feature detection. It then communicates
 the information to nfd-master which does the actual node labeling.  One
 instance of nfd-worker is supposed to be running on each node of the cluster,
 
-## Feature discovery
+## NFD-Topology-Updater
+
+NFD-Topology-Updater is a daemon responsible for examining allocated resources
+on a worker node to account for allocatable resources on a per-zone basis (where
```

Comment:
I think the term *allocatable* is a bit overloaded (wrt. node status for example). Should we use something like *available for new pods to be allocated* or similar here, WDYT?

Reply:
Sure, no problem. Will do.